override_values.tf

```diff
@@ -608,3 +608,29 @@ resources:
   EOT
   filename = "${path.module}/override_vales/kube_state_metrics.yaml"
 }
+
+
+resource "local_file" "keda_helm_config" {
+  count    = var.keda && (var.keda_helm_config == null) ? 1 : 0
+  content  = <<EOT
+global:  
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: "eks.amazonaws.com/nodegroup"
+            operator: In
+            values:
+            - "critical"
+  ## Using limits and requests
+  resources:
```

Comment:
there is no `resources:` field present as a direct parent of `limits:`, please fix this.

Reply:
done